x/commitment/keeper/msg_server_claim_kol.go

```diff
@@ -0,0 +1,79 @@
+package keeper
+
+import (
+	"context"
+
+	"cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/elys-network/elys/x/commitment/types"
+	ptypes "github.com/elys-network/elys/x/parameter/types"
+)
+
+var KolWallet = "elys1ykrphaysp9u5m8379f3d7pr9k7t0xq757lr9vj"
```

Comment:
@amityadav0 can you clarify why this address is hard coded?

Reply:
this address contains all liquid Elys, which will be used as a source to give to other addresses.
similar method is used in airdrop claim distribution

Reply:
I have verified that this is for main net

Reply:
that code will not work for testnet, so we need to identify the funding address for testnet and then use the chain id to use one address or the other according to chain id @amityadav0

Reply:
we will send the tokens to these addresses on testnet